Integrations/MicrosoftTeams/MicrosoftTeams.py

```diff
@@ -712,6 +712,100 @@ def get_users() -> list:
     return users.get('value', [])
 
 
+def add_user_to_channel(team_aad_id: str, channel_id: str, user_id: str):
+    """
+    Request for adding user to channel
+    """
+    url: str = f'{GRAPH_BASE_URL}/beta/teams/{team_aad_id}/channels/{channel_id}/members'
+    requestjson_: dict = {
+        '@odata.type': '#microsoft.graph.aadUserConversationMember',
+        'roles': [],
+        'user@odata.bind': f'https://graph.microsoft.com/beta/users/{user_id}'  # disable-secrets-detection
+    }
+    http_request('POST', url, json_=requestjson_)
+
+
+def add_user_to_channel_command():
+    """
+    Add user to channel (private channel only as still in beta mode)
+    """
+    channel_name: str = demisto.args().get('channel', '')
+    team_name: str = demisto.args().get('team', '')
+    member = demisto.args().get('member', '')
+    users: list = get_users()
+    user_id: str = str()
+    found_member: bool = False
+    for user in users:
+        if member in {user.get('displayName', ''), user.get('mail'), user.get('userPrincipalName')}:
+            found_member = True
+            user_id = user.get('id', '')
+            break
+    if not found_member:
+        demisto.results({
+            'Type': entryTypes['warning'],
+            'Contents': f'User {member} was not found',
+            'ContentsFormat': formats['text']
+        })
```

Comment:
Is this intentionally a warning and not an error? 
I think we should also check if `user_id` exists. Like this
`if not found_member or not iser_id:`